smartsim/_core/_cli/utils.py

```diff
@@ -33,18 +34,21 @@
 from smartsim._core.utils import colorize
 from smartsim.log import get_logger
 
-smart_logger_format = "[%(name)s] %(levelname)s %(message)s"
-logger = get_logger("Smart", fmt=smart_logger_format)
+SMART_LOGGER_FORMAT = "[%(name)s] %(levelname)s %(message)s"
+logger = get_logger("Smart", fmt=SMART_LOGGER_FORMAT)
 
 
 def get_install_path() -> Path:
-    try:
-        import smartsim as _
-    except (ImportError, ModuleNotFoundError):
+    module_spec = importlib.util.find_spec("smartsim")
+
+    if module_spec is None:
```

Comment:
You handled the same issue in SmartRedis differently (there, by disabling the lint check for non-top-of-file lint check).
While this change is very similar functionally, I'm not sure whether it's identical

Reply:
I did black-box testing with `smart clean` and `smart site` to ensure the same behavior. It was definitely a mitigation for the import location and I considered doing the same thing in SmartRedis. In SR, though, we alias and _use_ the import. Here, we were just letting exceptions control logic.

I'll do another sanity check here. Good eye

Reply:
here's a snippet showing matching outputs from the two different pieces of code.

```
import smartsim
import importlib

ss_path = smartsim.__path__[0]
il_path = importlib.util.find_spec("smartsim").submodule_search_locations[0]

print('ss_path', ss_path)
print('il_path', il_path)

# Output:
# ss_path /Users/ankona/code/ss/smartsim
# il_path /Users/ankona/code/ss/smartsim
```

Check out [this documentation](https://docs.python.org/3/library/importlib.html#:~:text=The%20purpose%20of%20the%20importlib,portable%20to%20any%20Python%20interpreter.) to assuage fears.